sdk-scheme-adapter/values.yaml

```diff
@@ -115,7 +115,7 @@ sdk-scheme-adapter-api-svc:
     QUOTES_ENDPOINT: '{{ .Release.Name }}-quoting-service'
     BULK_QUOTES_ENDPOINT: '{{ .Release.Name }}-quoting-service'
     TRANSACTION_REQUESTS_ENDPOINT: '{{ .Release.Name }}-transaction-requests-service'
-    TRANSFERS_ENDPOINT: '{{ .Release.Name }}-api-adapter-service'
+    TRANSFERS_ENDPOINT: '{{ .Release.Name }}-ml-api-adapter-service'
```

Comment:
Wow, how did that ever work :D

Reply:
I'm not sure but I picked this up in the MAS work when testing subscenarios.